### DISCUSSION TITLE
8.22 mining pools

### DISCUSSION THREAD

**Discussion by gonzoboards** [DigiByte-Core/digibyte#128] [General]
URL & Port for all 8.22 mining test pools, please! 🙏Thank You! 🏊‍♂️

**Comment by ycagel** [comment]
https://docs.google.com/spreadsheets/d/1PJyA1FxFot3M0thTEhDxjcX3eFfXPo9kuI3Y5zlGJY8/edit?usp=sharing

**Reply by gonzoboards** [reply]
Thanks @ycagel

**Reply by ycagel** [reply]
This came from Laura, so she is the one to thank!

**Reply by gonzoboards** [reply]
Nice! Thanks LT

**Comment by BlinddaveCrypto** [comment]
Learning site thru assistive technology...Idea to share I've had a few years on Merge Mining S Crypt
Thought Experiment...As Ltc has now resurrected Bel in Aug 2024, from where Doge was forked n now Merge Mined with most of S Crypt, I've witnessed the Hash of Bel increase drastically since implementation to match Ltc n Doge.
In theory, this would drastically increase the Hash of S Crypt for Dgb by utilizing the infinite supply of Doge to fund the tx beyond 2035, as Ltc has done for beyond 2150 tx's, as it is not profitable to mine Ltc or Dgb alone.
By merge mining S Crypt this will cause more miners to seek more profits, as with the only pool, Mining-Dutch.NL has done.
Thought is to ctrlA ctrlC ctrlV of open source, whilst changing a few parameters to include Dgb.
Another reason I purchased the old site DigiHash.co for a merge mining pool of S Crypt
Bel went from 60 Th/s to 827 Th/s overnight n currently at 1.78 Ph/s.
Theory would increase Hash of Dgb S Crypt from .25 to 1.2 Ph/s
I can articulate further, but new to GitHub n not sure of where to post...many ideas

I know that work is being done elsewhere, as priority,  n I don't wish to interrupt, but as a consideration for future use to fund tx beyond 2035.
Best Regards,
Blinddave

**Reply by BlinddaveCrypto** [reply]
PowerPool.io has listed many to merge mine, however Dgb is not there yet...would like to approach different pools to implement.

**Reply by nadav32p** [reply]
@ycagel @gto90 @JaredTate I know you guys have a lot going on with latest core protocol. However, Is this something that can be implemented? What do you think in terms of it being a viable option for miners? In terms of look, I think this will help with Digibyte's reputation and show more collaboration with a very popular coin. Just food for thought.

**Reply by gto90** [reply]
This topic has been raised by various community members over the years. I think significant risks would need to be worked through and debated in the community. Additionally, this would need to be proposed as a DIP.

The following are some topics that should be debated in a proposed DIP:

### 1. Increased Complexity and Implementation Risk

- **Protocol Modifications**: Adding merged mining requires changes at the consensus level (e.g., new rules for block headers) and in the mining software to handle AuxPoW data structures. This added complexity heightens the risk of protocol bugs or incompatibility issues.
- **Specialized Pool Support**: Mining pools must upgrade their infrastructure to merge-mine. If existing pools do not adopt these changes, actual hash-power gains might be minimal.

---

### 2. Potential Centralization of Mining Power

- **Reliance on a Larger Chain’s Miners**: Security depends on a handful of large mining pools on the primary chain. If these pools represent an overwhelming majority of the merge-mined hash rate, the smaller chain’s consensus could become highly centralized.
- **Inadequate Miner Incentives**: Since merge-mining adds minimal overhead for large miners, they may not prioritize the smaller chain’s security or governance if its rewards are insignificant compared to the primary chain. This can result in lower overall interest and potential neglect.

---

### 3. Upstream Dependency and Forced Upgrades

- **Protocol Changes on the Main Chain**: If the primary chain (on which you rely for merged mining) changes its Proof-of-Work algorithm, block structure, or other critical elements, your chain may be forced to make rapid, potentially disruptive upgrades to remain compatible.
- **Risk of Hash-Rate Loss**: Any significant policy change—or even a contentious fork—on the main chain can lead to a sudden drop in merge-mined hash rate, reducing the smaller chain’s security unexpectedly.

---

### 4. Attack Vectors and Exploits

- **51% Attacks**: While merged mining generally increases the hash rate available to the smaller chain, it also makes a well-capitalized miner on the primary chain capable of reorganizing blocks on the smaller chain at relatively little extra cost.
- **Double-Spend Risks**: A miner that already commands high hash power on the primary chain can secretly build alternative chains on the smaller network and execute double-spend attacks with reduced financial risk compared to launching an attack from scratch.

---

### 5. Ecosystem and Governance Concerns

- **Influence from External Miners**: Large pools from the main chain may exert disproportionate influence on governance if they decide to coordinate (e.g., refusing to mine certain upgrades).
- **Community Perception**: Some communities prefer standalone security over reliance on a larger blockchain. Aligning incentives and addressing perceived “dependency” can pose political and social challenges.

---

### 6. Pool Adoption and Coordination

- **Slow Adoption Curve**: Merged mining requires mining pools to invest time and resources into development and testing. If major pools do not adopt, the smaller chain sees little benefit.
- **Operational Overhead**: Pool operators must handle payouts, track extra rewards, and ensure correct validation for both chains. Mistakes can lead to orphaned blocks, payout disputes, or additional attack vectors.

---

### 7. Economic Considerations

- **Block Reward Competitiveness**: The smaller chain needs a sufficiently attractive block reward or fee structure to entice miners. If rewards are too small, pools may not bother upgrading their software to merge-mine.
- **Market Volatility**: Merged miners effectively hold rewards from two chains, exposing them to more complex market conditions. If the smaller chain’s token or environment becomes volatile, miners may withdraw support quickly.

---
